lib/src/main/java/growthbook/sdk/java/ExperimentResult.java

```diff
@@ -1,57 +1,96 @@
 package growthbook.sdk.java;
 
 import com.google.gson.JsonElement;
-import com.google.gson.JsonObject;
-import com.google.gson.JsonSerializationContext;
-import com.google.gson.JsonSerializer;
-import lombok.AllArgsConstructor;
+import com.google.gson.annotations.SerializedName;
 import lombok.Builder;
 import lombok.Data;
 
 import javax.annotation.Nullable;
-import java.lang.reflect.Type;
 
 /**
- * The result of running an {@link Experiment} given a specific {@link GBContext}
- *
- * <ul>
- * <li>inExperiment (boolean) - Whether or not the user is part of the experiment</li>
- * <li>variationId (int) - The array index of the assigned variation</li>
- * <li>value (any) - The array value of the assigned variation</li>
- * <li>hashUsed (boolean) - If a hash was used to assign a variation</li>
- * <li>hashAttribute (string) - The user attribute used to assign a variation</li>
- * <li>hashValue (string) - The value of that attribute</li>
- * <li>featureId (string or null) - The id of the feature (if any) that the experiment came from</li>
- * </ul>
- *
+ * The result of an {@link GrowthBook#run(Experiment)} call
  * @param <ValueType> generic type for the value type for this experiment's variations.
  */
 @Data
-@Builder
-@AllArgsConstructor
```

Comment:
Moved from this property-based approach to the explicit `@Builder`-annotated constructor.